### ISSUE TITLE
docker images is a great alias for docker image ls, while docker volumes ls does not exist

### ISSUE THREAD

**Issue by rafaelucena** [docker/cli#4600]
### Description

Using docker for a few years now, every few months or years I like to do a cleanup on the volumes and images and start over, and it keeps bugging me that for showing the images we can run `docker images` but for volumes we can't use `docker volumes`. I'm aiming to fix that.

When having at least one volume:

```shell
docker volumes
```

### Expected behavior

The output should be the same as:

```shell
docker volume ls
```

### docker version

(not strictly necessary?)

### docker info

(not strictly necessary?)

### Additional Info

This is my first issue, please be kind (sorry)

**Comment by nicks** [comment]
i think `docker images` is considered a legacy command. it's there for backwards compatibility and we generally discourage it. see this great comment! https://github.com/docker/cli/issues/887#issuecomment-366654909

**Comment by rafaelucena** [comment]
Yeah, @thaJeztah mentioned it on the PR and he explained it in another issue from another user also. It's a shame really, my favourite command is expect to depart :(

Well, it is what it is. Oh well, I'll probably just handle some bash aliases locally for that :)

RIP.

**Comment by thaJeztah** [comment]
Yes, closing this one, because this was by design.
